js/apps/admin-ui/src/i18n/i18n.ts

```diff
@@ -6,40 +6,17 @@ import { joinPath } from "../utils/joinPath";
 import { OverridesBackend } from "./OverridesBackend";
 
 export const DEFAULT_LOCALE = "en";
-export const DEFAULT_NAMESPACE = "common";
+export const DEFAULT_NAMESPACE = "translation";
 export const NAMESPACE_SEPARATOR = ":";
```

Comment:
This one is only used for the overrides back-end. The overrides backend should be refactored, so that it no longer uses namespaces. Otherwise the realm overrides will break. Once that has been done, this variable can be removed.

```suggestion
```